src/components/notifications/notifications-list.tsx

```diff
@@ -0,0 +1,207 @@
+import { useGraphqlSwr } from '@/api/use-graphql-swr'
+import {
+  NotificationEvent,
+  NotificationEventPayload,
+  NotificationEventType,
+  parseNotificationEvent,
+} from '@/events/event'
+
+export function NotificationsList() {
+  const { data } = useGraphqlSwr<{
+    notifications: {
+      nodes: {
+        id: number
+        unread: boolean
+        event: NotificationEventPayload
+      }[]
+    }
+  }>({
+    query: `
+      query notifications($count: Int!, $unread: Boolean){
+        notifications(first: $count, unread: $unread) {
+          nodes {
+            id
+            unread
+            event {
+              type
+              date
+              actor {
+                id
+                username
+              }
+              object {
+                id
+              }
+              payload
```

Comment:
@inyono could we get the title of the Entity/Page/Repository here as well as the id?
This would be needed to replicate current behaviour (which is nice).

Reply:
(my understanding is, that this would have to be changes in the api, right?)

Reply:
See Slack PN, let's talk about that before I make any changes.